src/wagtail_ai/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from .prompts import Prompt as PromptDataClass
+
+
+class Prompt(models.Model):
```

Comment:
**Thought:** It may be beneficial to have an admin interface title, to distinguish between different versions of the same prompt.